### PR TITLE
[AF-1074]: Removed unused OrganizationalUnitManager View that made connected clients crash.

### PR DESCRIPTION
**Pull request by adrielparedes**
https://github.com/kiegroup/appformer/pull/247
https://github.com/kiegroup/kie-wb-common/pull/1511
https://github.com/kiegroup/kie-wb-distributions/pull/709
https://github.com/kiegroup/drools-wb/pull/832

### CODE REVIEW

File: kie-drools-wb-parent/kie-drools-wb-webapp/pom.xml [hunk 1]
```diff
@@ -1846,13 +1841,12 @@
...
               <compileSourcesArtifact>org.uberfire:uberfire-message-console-api</compileSourcesArtifact>
               <compileSourcesArtifact>org.uberfire:uberfire-message-console-client</compileSourcesArtifact>
-              <compileSourcesArtifact>org.uberfire:uberfire-organizationalunit-manager</compileSourcesArtifact>
 
               <!-- Guvnor ALA -->
               <compileSourcesArtifact>org.kie.workbench:kie-wb-common-ala-ui-api</compileSourcesArtifact>
               <compileSourcesArtifact>org.kie.workbench:kie-wb-common-ala-ui-client</compileSourcesArtifact>
               <compileSourcesArtifact>org.kie.workbench:kie-wb-common-ala-openshift-ui-api</compileSourcesArtifact>
-              <compileSourcesArtifact>org.kie.workbench:kie-wb-common-ala-openshift-ui-client</compileSourcesArtifact>
+              <compileSourcesArtifact>org.kie.workbenc:kie-wb-common-ala-openshift-ui-client</compileSourcesArtifact>
```
Comment (ederign) [inline kie-drools-wb-parent/kie-drools-wb-webapp/pom.xml]: ?

**Comment by adrielparedes** [comment]
jenkins retest this

**Comment by ederign** [comment]
jenkins retest this

**Comment by mbiarnes** [comment]
Jenkins please retest this

**Comment by adrielparedes** [comment]
@mbiarnes @ederign Am I the only one who has this tests failing? I made smoked it and it worked, but this tests are not. Also I didn't saw this tests has relation with this PR

**Comment by jhrcek** [comment]
@adrielparedes FYI 4 of those failures are selenium tests, which are failing deterministically on master - I submitted [PR|https://github.com/kiegroup/kie-wb-distributions/pull/712] to fix those. However I don't know about the failing REST tests. I believe @cristianonicolai has  [PR|https://github.com/kiegroup/kie-wb-distributions/pull/711] in progress to fix those..

**Comment by adrielparedes** [comment]
@jhrcek oh! good to know, may be @cristianonicolai could help us to check if that failing test is resolved in that PR or if it is my mistake so we can merge it and fix master.

**Comment by jhrcek** [comment]
@adrielparedes let's rebase on master and test again, please.

**Comment by adrielparedes** [comment]
@jhrcek rebased, waiting for tests

**Comment by cristianonicolai** [comment]
Tests on kie-wb-tests (rest and selenium) still failing randomly due to conflicts with running two Wildfly instances in parallel. Trying to fix that in another PR https://github.com/kiegroup/kie-wb-distributions/pull/711

**Comment by ederign** [comment]
yaaay!

**Pull request merged**